_includes/tutorials/naming-changelog-repartition-topics/kstreams/code/build.gradle

```diff
@@ -0,0 +1,63 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "com.commercehub.gradle.plugin:gradle-avro-plugin:0.15.1"
+        classpath "com.github.jengelman.gradle.plugins:shadow:4.0.2"
+    }
+}
+
+plugins {
+    id "java"
+    id "com.google.cloud.tools.jib" version "1.1.1"
+    id "idea"
+    id "eclipse"
+}
+
+sourceCompatibility = "1.8"
+targetCompatibility = "1.8"
+version = "0.0.1"
+
+repositories {
+    mavenCentral()
+    jcenter()
+
+    maven {
+        url "http://packages.confluent.io/maven"
+    }
+}
+
+apply plugin: "com.commercehub.gradle.plugin.avro"
+apply plugin: "com.github.johnrengelman.shadow"
+
+dependencies {
+    compile "org.apache.avro:avro:1.8.2"
```

Comment:
```suggestion
    implementation "org.apache.avro:avro:1.8.2"
```

Reply:
also it's a good idea to update avro thingy. we support Avro 1.9 in cp 5.4.1

Reply:
ack. will do